esmvaltool/recipes/ref/recipe_ref_scatterplot.yml

```diff
@@ -0,0 +1,317 @@
+# ESMValTool
+---
+documentation:
+  title: Model evaluation with focus on clouds.
+  description: >
+    Plot 2D histograms using the Seaborn diagnostic in ESMValTool.
+  authors:
+    - bock_lisa
+    - lauer_axel
+  maintainer:
+    - bock_lisa
+
+timerange_for_data: &time_period
+  timerange: '2007/2011'  # can be specified, this is just an example
+regridding: &regridding
+  regrid:
+    target_grid: 1x1
+    scheme:
+      reference: esmf_regrid.schemes:ESMFAreaWeighted
+levels_for_3d_vars: &extracting_levels
+  extract_levels:
+    levels: {cmor_table: CMIP6, coordinate: plev27}
+    coordinate: air_pressure
+    scheme: linear
+
+
+preprocessors:
+  common_grid_clt:
+    <<: *regridding
+    custom_order: true
+    mask_below_threshold:
+      threshold: 0.001
+    mask_above_threshold:
+      threshold: 100.
```

Comment:
We also have preprocessor [`mask_outside_range`](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/api/esmvalcore.preprocessor.html#esmvalcore.preprocessor.mask_outside_range) that combines these two.

Reply:
Thanks! I changed that!